### PR TITLE
Better separation between provisioning and services

### PR DESCRIPTION
**Pull request by mogul**
...plus introductory material about deployer accounts and some minor text cleanups.

### CODE REVIEW

File: content/docs/apps/continuous-deployment.md [hunk 1]
```diff
@@ -51,7 +53,7 @@ $ cf delete-service my-deployer-account
 
 ### *East/West environment:* Ask an admin
 
-The first thing you need to do is ask an admin to setup a "deployer" user in your organization and give it permission to deploy to the desired space:
+Ask an admin to setup a "deployer" user in your organization and give it permission to deploy to the desired space:
```
Comment (brittag) [inline content/docs/apps/continuous-deployment.md]: This is a good opportunity to address https://github.com/18F/cg-site/issues/453 - maybe just "Ask cloud.gov support to set up a "deployer" user in your organization and give it permission to deploy to the desired space." and delete the commands? (Presumably we know the command to use or have it documented elsewhere.)

**Comment by brittag** [comment]
This looks good to me from a content perspective - just added another suggestion. I'll leave it to another person to review for technical accuracy though.

**Comment by mogul** [comment]
@brittag I did some surgery since this one was stale. I took your suggestion and don't think there's any technical content here for review by others... Want to review/accept now?

File: content/docs/apps/continuous-deployment.md [hunk 2]
```diff
@@ -52,14 +54,14 @@ $ cf delete-service my-deployer-account
 {{% eastwest %}}
 ### Ask an admin
 
-The first thing you need to do is ask an admin to setup a "deployer" user in your organization and give it permission to deploy to the desired space:
+Ask an admin to setup a "deployer" user in your organization and give it permission to deploy to the desired space.
```
Comment (brittag) [inline content/docs/apps/continuous-deployment.md:57]: Should this be [Ask support](https://cloud.gov/docs/help/) rather than ask an admin?

**Comment by brittag** [comment]
Yay, this is better than before and solves a problem, thanks!

**Review by brittag — APPROVED** [review]

**Pull request merged**